extensions/interactions/NumericInput/directives/NumericInput.js

```diff
@@ -65,6 +65,19 @@ oppia.directive('oppiaInteractiveNumericInput', [
           $scope.$on(EVENT_PROGRESS_NAV_SUBMITTED, function() {
             $scope.submitAnswer($scope.answer);
           });
+
+          $scope.isAnswerValid = function() {
+            return ($scope.answer !== undefined && 
```

Comment:
Style: break after '(' if a parenthesized clause spans multiple lines.

Reply:
Will remember that. Done.